packages/docs/src/stories/components/tag.stories.ts

```diff
@@ -51,13 +51,28 @@ export const Size = {
 
 /**
  * Use the `selected` attribute to enable the selected state.
+ *
+ * __Accessibility hint__: Use the attribute `toggleable` to reflect the `selected` state on the `aria-pressed` attribute.
  */
 
 export const Selected = {
   render: () => html`
-    <div class="flex gap-12">
-      <sd-tag selected>Selected</sd-tag>
+    <div id="tags-selected" class="flex gap-12">
+      <sd-tag selected toggleable>Selected</sd-tag>
+      <sd-tag toggleable>Unselected</sd-tag>
     </div>
+
+    <script>
```

Comment:
```suggestion
    <script type="module">
```

If you change to another component and back to the sd-tag, you get an error that `tag` and `handleToggle` is already been declared. Adding `type="module"` fixes the issue.